crates/wkg/src/main.rs

```diff
@@ -19,18 +23,22 @@ struct Cli {
 #[derive(Args, Debug)]
 struct RegistryArgs {
     /// The registry domain to use. Overrides configuration file(s).
-    #[arg(long = "registry", value_name = "DOMAIN")]
+    #[arg(long = "registry", value_name = "REGISTRY", env = "WKG_REGISTRY")]
     registry: Option<Registry>,
 }
 
 #[derive(Subcommand, Debug)]
+#[allow(clippy::large_enum_variant)]
 enum Commands {
-    /// Get a package.
-    Get(GetCommand),
+    /// Load a package. This is for use in debugging dependency fetching. For pulling a component, use `wit get`
+    Load(LoadArgs),
```

Comment:
I don't feel too strongly about `get` vs `load`, though `get` is at least familiar to go devs. :shrug:

Reply:
Leaning towards keeping it as `wkg get` since it is short and easier to remember (at least for me).

Reply:
Ok, I'll swap that out

Reply:
Changed it to get!